packages/virtual-core/src/index.ts

```diff
@@ -341,7 +355,7 @@ export class Virtualizer<TScrollElement = unknown, TItemElement = unknown> {
 
       this.unsubs.push(
         this.options.observeElementOffset(this, (offset) => {
-          this.scrollOffset = offset
+          this.scrollOffset = offset;
```

Comment:
did you mean to add this `;`?

Seems like the use of ; is pretty inconsistent throughout